### PR TITLE
Integration contribution for SpyCloud ATO

### PR DESCRIPTION
**Pull request by nheijmans**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Contributing xsoar integration for SpyCloud API ATO

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by nheijmans** [comment]
@ilaner hi 👋🏻 ! The validation error https://app.circleci.com/pipelines/github/demisto/content/144715/workflows/547aadc3-7284-4d1b-96d8-0578f8a2e0a8/jobs/481797?invite=true#step-108-58 here is unclear to me. Could you help identify what the check wants from us :) ?

**Comment by Noy-Maimon** [comment]
> @ilaner hi 👋🏻 ! The validation error https://app.circleci.com/pipelines/github/demisto/content/144715/workflows/547aadc3-7284-4d1b-96d8-0578f8a2e0a8/jobs/481797?invite=true#step-108-58 here is unclear to me. Could you help identify what the check wants from us :) ?

Hi @nheijmans, so sorry for the late reply. Thank you for your contribution. I had a look at the error you specified and it seems to be like a temporary issue. can you please try pushing the branch again? this way the build will be triggered again, and we can see if this error reoccures. Thanks!

File: Packs/SpyCloudPack/Integrations/SpyCloud/SpyCloud.py [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+
+    if key in spydata.keys():
+        transformed_data = spydata[key]
+    else:
+        transformed_data = "empty"
+
+    return transformed_data
+
+
+def test_mod():
```
Comment (Noy-Maimon) [inline Packs/SpyCloudPack/Integrations/SpyCloud/SpyCloud.py:256]:
```suggestion
def test_module():
```

File: Packs/SpyCloudPack/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# SpyCloud Integration
+Integration for the SpyCloud ATO service. Retrieve breach data for
+- Domain searches
+- Email searches
+- Breaches
+
+Filter arguments are added as well. 
+
+**Make sure to whitelist your outbound IP address XSOAR uses to connect**
```
Comment (Noy-Maimon) [inline Packs/SpyCloudPack/README.md:9]:
```suggestion
```

When the pack is only of an integration (as in this case), we can use only the integration readme (no need for duplicate).

**Review by Noy-Maimon — CHANGES_REQUESTED** [review]
Looks great!
Please note my comments and change where needed.

File: Packs/SpyCloudPack/README.md (hunk 2 above)
Comment (Noy-Maimon) [inline Packs/SpyCloudPack/README.md:9]: Please modify the Pack readme to meet our standards:
https://xsoar.pan.dev/docs/documentation/pack-docs#readme-structure

File: Packs/SpyCloudPack/Integrations/SpyCloud/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# SpyCloud Integration
+Integration for the SpyCloud ATO service. Retrieve breach data for
+- Domain searches
+- Email searches
+- Breaches
+
+Filter arguments are added as well.
+
+**Make sure to whitelist your outbound IP address XSOAR uses to connect**
```
Comment (Noy-Maimon) [inline Packs/SpyCloudPack/Integrations/SpyCloud/README.md:9]: Please modify the **Integration** readme to meet our standards:
https://xsoar.pan.dev/docs/documentation/readme_file

**Review by nheijmans — COMMENTED** [review]
Thanks for those additional changes! Tested and seems to work for me

**Comment by Noy-Maimon** [comment]
@nheijmans Hi, here's what you have left:
1. to solve the secrets issue - add this email to the .secrets-ignore file: john.doe@hotmail.com
2. modify the Pack README (as mentioned in the comment on the README file [here](https://github.com/demisto/content/pull/17604/files/e6a844ede9ef723c9412b40cff5e16008ccb1050#diff-99ff0b77ee7043888c956ce13a21572904b4dc0834c1e0fa8f164b2ad77daf35))
3. merge your forked repo from our content master and then merge it to your branch. let me know if you need assistance with that.

**Comment by nheijmans** [comment]
> 2\. here

@Noy-Maimon I've pushed the secret ignore change
On 2, I'm not sure what you mean, the docs have been generated with demisto-sdk generate-docs, they are the standard as documented

On 3, yes please give a hand with this :) I'm not entirely sure whats required here

**Comment by Noy-Maimon** [comment]
> > 2. here
> 
> @Noy-Maimon I've pushed the secret ignore change On 2, I'm not sure what you mean, the docs have been generated with demisto-sdk generate-docs, they are the standard as documented
> 
> On 3, yes please give a hand with this :) I'm not entirely sure whats required here

You are talking about the Integration README (Packs/SpyCloudPack/Integrations/SpyCloud/README.md), I was referring to the **pack** readme (Packs/SpyCloudPack/README.md). 

Each pack readme should contain:

- A short paragraph connecting real-life situations to the pack use cases.
- A "What does this pack do?" section, explaining point-by-point the capabilities of the pack or the main playbook of the pack.
- (Optional): a sentence or two detailing the contents of the pack.
See more information [here](https://xsoar.pan.dev/docs/documentation/pack-docs#readme-structure).

**Pull request merged**